nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -294,6 +294,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 - A new module was added for the [Starship](https://starship.rs/) shell prompt,
   providing the options `programs.starship.enable` and `programs.starship.settings`.
 
+- The [Dino](https://dino.im) XMPP client was updated to 0.3, adding support for audio and video calls.
```

Comment:
This is a none breaking change which only a very small group of people cares about. Why do we need a changelog entry?

Reply:
Well, idk. The changelog file is more a giant mess. Fine with me if you want to keep it.